src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -69,155 +71,44 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln(sprintf('Clearing the cache for the <info>%s</info> environment with debug <info>%s</info>', $kernel->getEnvironment(), var_export($kernel->isDebug(), true)));
         $this->getContainer()->get('cache_clearer')->clear($realCacheDir);
 
-        if ($input->getOption('no-warmup')) {
-            $filesystem->rename($realCacheDir, $oldCacheDir);
-        } else {
-            // the warmup cache dir name must have the same length than the real one
-            // to avoid the many problems in serialized resources files
-            $warmupDir = substr($realCacheDir, 0, -1).'_';
+        $filesystem->rename($realCacheDir, $oldCacheDir);
```

Comment:
Renaming the cache after the warmup rather than before was done to minimize downtime for other processes accessing the cache AFAIK.